manifests/01-registry-credentials-request-powervs.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: openshift-image-registry-ibmcos
```

Comment:
`manifests/01-registry-credentials-request-ibmcos.yaml` already uses that name.

Reply:
renamed.